network/src/protocols/ping.rs

```diff
@@ -244,6 +258,40 @@ impl ServiceProtocol for PingHandler {
             _ => panic!("unknown token {}", token),
         }
     }
+
+    fn poll(
+        mut self: Pin<&mut Self>,
+        cx: &mut Context,
+        context: &mut ProtocolContext,
+    ) -> Poll<Option<()>> {
+        self.control_receiver
+            .poll_next_unpin(cx)
+            .map(|control_message| {
+                control_message.map(|_| {
+                    let now = SystemTime::now();
+                    let peers: Vec<SessionId> = self
+                        .connected_session_ids
+                        .iter_mut()
```

Comment:
https://github.com/nervosnetwork/ckb/pull/2257/files#diff-d189679dd379fe6aac5b2b82d73c9e22L181

must filter processing peer，other wise it will be disconnected

Reply:
😬  resolved, please review again, thanks